app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -30,6 +40,9 @@ interface ThreadViewModel {
 
         private val rootComment = BehaviorSubject.create<Comment>()
         private val focusOnCompose = BehaviorSubject.create<Boolean>()
+        private val currentUserAvatar = BehaviorSubject.create<String?>()
+        private val replayComposerStatus = BehaviorSubject.create<CommentComposerStatus>()
+        private val showReplayComposer = BehaviorSubject.create<Boolean>()
```

Comment:
nit: change to `replyComposerStatus` and `showReplyComposer`

Reply:
Done